app/src/main/java/ru/vssemikoz/newsfeed/models/ShowOnlyFavorite.java

```diff
@@ -0,0 +1,25 @@
+package ru.vssemikoz.newsfeed.models;
+
+public enum ShowOnlyFavorite {
```

Comment:
Something happened with the code style and empty lines here. BTW, why do we need this enum? Can't we invert by simply doing this: `showFavorite = !showFavorite`?